elasticsearch5/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,7 @@
+appender.console.type = Console
```

Comment:
Not sure what is getting logged by this but if it is during the YCSB testing should go to stderr instead of stdout.